code/modules/projectiles/boxes_magazines/external/rechargable.dm

```diff
@@ -114,3 +115,9 @@
 	type_color = "#d084d6"
 	type_name = "<span class='ion'>ION</span>"
 	projectile_type = /obj/item/projectile/ion
+
+/obj/item/ammo_casing/mws_batt/taser
+	name = "'MWS' microbattery - TASER"
+	type_color = "#e5ff00"
+	type_name = "<span class='stun'>TASE</span>"
+	projectile_type = /obj/item/projectile/energy/electrode
```

Comment:
is this not the electrode projectile that knocks people down and was removed from the HoS gun because it was the only ranged knockdown security had which made it blatantly overpowered compared to Literally Everything Else in the arsenal
pretty sure the balanced one is /obj/item/projectile/energy/electrode/security but the security electrode is just a slowdown so why would you ever use that over the disabler cartridge unless you totally knew what this was and wanted to make it Substantially Better than the multiphase

Reply:
I totally didnt know what I was doing but the reason is literally tase > lase, I'll swap it out with the not funny electrode